### PR TITLE
App translate10 - home page fr

### PR DESCRIPTION
**Pull request by zerifah**

### CODE REVIEW

**Comment by zerifah** [comment]
The video is on Teams

**Comment by zerifah** [comment]
I've add the videos. I don't have do a new PR, it's ok? (It would be take the new commit?)

File: src/pages/index.tsx [hunk 1]
```diff
@@ -487,14 +487,6 @@ const Home = observer(() => {
                         </div>
                     ) : (
                         <div className="card" style={{ boxShadow: 'var(--ifm-global-shadow-md)' }}>
```
Comment (lebalz) [inline src/pages/index.tsx:489]:
```suggestion
                        <div className="card" style={{ boxShadow: 'var(--ifm-global-shadow-md)' }}>
                          <div className="card__header">
                                <h2>Phase d'introduction</h2>
                            </div>
                            <div className="card__body">
                                Le calendrier est encore dans sa phase d'introduction. Des retours peuvent être transmis
                                grâce à l'application de traçage de bugs (voir Video).
                                <h3>Merci beaucoup pour votre aide.</h3>
                            </div>
```

Comment (lebalz) [reply src/pages/index.tsx:489]: i'd keep this section to make sure users are aware of tha fact that the calendar is still in a introductory phase...

**Review by lebalz — CHANGES_REQUESTED** [review]

**Review by lebalz — APPROVED** [review]

**Pull request merged**